app/services/weets.js

```diff
@@ -0,0 +1,18 @@
+const axios = require('axios');
+const logger = require('../logger');
+const { clientAPI, limit } = require('../../config/index.js').common;
+
+exports.getRandomQuote = async () => {
+    try {
+        const response = await axios.get(clientAPI, {
+            params: {
+                limit: limit
+            }
+        });
+        const { data } = response.data;
+        return data[0];
```

Comment:
Here you can return all the elements of the array and you can also try to do a destructuring of the API response.